news/_posts/releases/2020-12-03-prestashop-1-7-7-0-available.md

```diff
@@ -0,0 +1,213 @@
+- [An exception is displayed in the console when adding an out of stock product](https://github.com/PrestaShop/PrestaShop/issues/21794)
+- [An exception is displayed when canceling a product and if an invoice is generated](https://github.com/PrestaShop/PrestaShop/issues/21850)
+- [An exception is displayed when ordering a product with a file customization’s name containing a space](https://github.com/PrestaShop/PrestaShop/issues/21815)
+- [When all the products in an order are deleted, the status is changed to canceled.](https://github.com/PrestaShop/PrestaShop/issues/22010)
+- [When deleting all products the invoice's total is wrong](https://github.com/PrestaShop/PrestaShop/issues/21991)
+- [When we have a product added to the order as a gift (a voucher applied) and as a product ordered, the quantity added to the same product is wrong](https://github.com/PrestaShop/PrestaShop/issues/21531)
+- [When product is added in an empty order, the tax block in the invoice is wrong: It only contains the Shipping tax and not the Product tax](https://github.com/PrestaShop/PrestaShop/issues/22008)
+- [An error is displayed when decreasing the gifted product quantity "Could not find the product in cart, meaning Order and Cart are out of sync"](https://github.com/PrestaShop/PrestaShop/issues/22004)
+- [When adding a product which has an ecotax to an old order, the base price is NOK](https://github.com/PrestaShop/PrestaShop/issues/22029)
+
+**Back-office**
+- [After logging in to the Addons marketplace, the list of recommended modules doesn't contain all items](https://github.com/PrestaShop/PrestaShop/issues/22125)
+
+**Front-office**
+- [Logo, favicon and store's icons are not displayed](https://github.com/PrestaShop/PrestaShop/issues/21979)
+- [Add to cart Popup not displayed after an upgrade](https://github.com/PrestaShop/PrestaShop/issues/22173)
+
+**Upgrade**
+- [Error "Column count doesn't match value count" during the upgrade](https://github.com/PrestaShop/PrestaShop/issues/21949)
+- [Problem with ALTER table during the upgrade](https://github.com/PrestaShop/PrestaShop/issues/21948)
+- [Error when upgrading from 1.7.4.x and previous to 1.7.7.0](https://github.com/PrestaShop/PrestaShop/issues/22141)
+- [After an upgrade the precision of each currency installed is set to 6](https://github.com/PrestaShop/PrestaShop/issues/22177)
+
+## Changelog
+
+24 pull requests have been merged since the RC1 and 22 issues have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. This brings us to more than 1300 merged pull requests for the 1.7.7.0 milestone, it is definitely our biggest yet.
+
+If you are looking for more details about all changes and new features expected in 1.7.7, the [1.7.7 beta version release note](https://build.prestashop.com/news/prestashop-1-7-7-0-beta-release/) is the perfect article for you to read!
+
+## Auto upgrade
```

Comment:
@marionf @eternoendless Here is the note about the Autoupgrade required version, do you think it's enough?

Reply:
(besides the `you` which needs to be fixed into `your`)